src/nvim/drawline.c

```diff
@@ -441,7 +442,7 @@ void fill_foldcolumn(win_T *wp, foldinfo_T foldinfo, linenr_T lnum, int attr, in
 static void draw_sign(bool nrcol, win_T *wp, winlinevars_T *wlv, int sign_idx, int sign_cul_attr)
 {
   SignTextAttrs sattr = wlv->sattrs[sign_idx];
-  wlv->c_final = NUL;
+  wlv->sc_final = NUL;
```

Comment:
No longer used right for drawing the columns right? Neither is `wlv->sc_extra` below.
```suggestion
```

Reply:
yes, thanks.